app/qml/main.qml

```diff
@@ -238,6 +239,9 @@ ApplicationWindow {
         target: mapCanvas.mapSettings
         onExtentChanged: {
             scaleBar.visible = true
+            var pos = mapCanvas.mapSettings.coordinateToScreen( positionKit.projectedPosition)
+            positionMarker.animatedPosX = pos.x
+            positionMarker.animatedPosY = pos.y
```

Comment:
Rather than configuring some internal properties of position marker here, this should go to PositionMarker implementation so that all the positioning code is in one place